client/src/pages/private/Admin.js

```diff
@@ -51,33 +53,26 @@ export default () => {
 
   return (
     <Page >
-      <Grid container alignContent="center" justify="center" alignItems="center" direction="column">
-        <Box pb={4} pl={4} pr={4} pt={2}>
-          <Grid container direction="column">
-            {
-              roles.length === 0 ?
-                <Typography variant="subtitle1" gutterBottom>
-                  You don't have enough permissions.
+      <CheckPermissions roles={roles} permittedRoles={['maximus', 'employer', 'health_authority', 'ministry_of_health']} renderMessage={true}>
```

Comment:
I think in the next PRs we should centralize those role names constants, I'm starting to feel the pain of changing string occurrences around the code xD

Reply:
Yeah I was trying to figure out a good way to manage constants in general, since we have so many that need to be accessed by both the FE and BE... It seems very unwieldy to pass them in via env/make/docker since they aren't secrets or config.